core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -309,7 +310,9 @@ void createTableInternal(
     // Add compaction strategy
     CompactionStrategy compactionStrategy =
         CompactionStrategy.valueOf(
-            options.getOrDefault(COMPACTION_STRATEGY, CompactionStrategy.STCS.toString()));
+            options
+                .getOrDefault(COMPACTION_STRATEGY, CompactionStrategy.STCS.toString())
+                .toUpperCase(Locale.ROOT));
```

Comment:
Use `toUpperCase()` so that uses can use both lower case and upper case for the configuration.